ui/ui-components/pages/_app.tsx

```diff
@@ -108,10 +114,12 @@ GrouparooNextApp.getInitialProps = async (appContext: AppContext) => {
       navigation: navigationResponse.navigation,
       clusterName: navigationResponse.clusterName,
       hydrationError,
+      client,
     };
   } catch (_error) {
     const { formattedErrorMessage, formattedErrorObject } = renderError(_error);
     return {
+      client,
```

Comment:
To confirm: This client will only be used the first time the page renders on the server side?

Reply:
I'll be pushing up a slight logic change. As long as this client is a client and not some serialized object, it will be used.

Reply:
It will always get a serialized object when the browser side runs. I'm wondering if we ever need to share this instance with the page server side.